README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
+| Pruning of builds     |       |      |    |
```

Comment:
this is suppose to be a tekton feature (though if it has landed I have missed it)

perhaps an opportunity for either DEVEX or APPSVC to contribute upstream @adambkaplan @siamaksade @bparees @sbose78

Reply:
https://github.com/redhat-developer/build/issues/53